### ISSUE TITLE
swiftlint reports wrong version

### ISSUE THREAD

**Issue by jberkel** [realm/SwiftLint#3887]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

The version returned by `swiftlint version` is incorrect, and does not correspond to the downloaded package.

```sh
$ curl -L https://github.com/realm/SwiftLint/releases/download/0.46.5/SwiftLint.pkg > SwiftLint.pkg
$ sudo installer -pkg SwiftLint.pkg -target /
$ swiftlint version
0.46.4
```

(The 0.46.4 pkg reports version 0.46.3)

Update: it looks like swiftlint gets installed twice, to `/usr/local/bin/swiftlint` and `/usr/local/bin/usr/local/bin/swiftlint`. The latter binary has the correct version. (I made sure to `rm /usr/local/bin/swiftlint` before running the installation)

Looks related to #3815

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?

0.46.5 (4)

* Installation method used (Homebrew, CocoaPods, building from source, etc)?

manual, Github releases

**Comment by jpsim** [comment]
To be honest I didn't believe you, so I installed the package in a VM and you're absolutely right, the package does install SwiftLint twice:

<img width="574" alt="image" src="https://user-images.githubusercontent.com/474794/157752682-9f77b883-657d-4d2e-b111-18525259b077.png">

Here's where the package is created: https://github.com/realm/SwiftLint/blob/3273cef42b17a350aa993f5fb311ec03901ef89a/Makefile#L114-L120

The `installables` make command must be incorrect: https://github.com/realm/SwiftLint/blob/3273cef42b17a350aa993f5fb311ec03901ef89a/Makefile#L94-L96

**Comment by jberkel** [comment]
Thanks, in the meantime I switched to the portable zip version, which can be installed to any location and does not require root permissions.
